snowpack/src/index.ts

```diff
@@ -12,11 +12,22 @@ import {CLIFlags} from './types/snowpack';
 import {clearCache, readLockfile} from './util.js';
 export * from './types/snowpack';
 
-// Unstable: These APIs are in progress and subject to change
-export {startServer as unstable__startServer} from './commands/dev';
-export {loadAndValidateConfig as unstable__loadAndValidateConfig};
+// Stable API:
+export {startServer, ServerResult} from './commands/dev';
+export {loadAndValidateConfig};
+
+// Unstable: These APIs are in progress and subject to change across minor versions
 export {getUrlForFile as unstable__getUrlForFile} from './build/file-urls';
 
+/** DEPRECATED: Promoted to startServer() **/
```

Comment:
General note: if you make this `/** @deprecated Promoted to startServer() **/` then it shows up in the editor with a ~line through~

Reply:
great tip, thanks!